app/gui2/src/components/DocumentationPanel/DocsTag.vue

```diff
@@ -0,0 +1,31 @@
+<script setup lang="ts">
+import { computed, ref, watch } from 'vue'
+import type { View } from './DocsTags.vue'
+
+const props = defineProps<{ view: View; parentOffset: number; forceShow: boolean }>()
+const emit = defineEmits<{ hidden: [] }>()
+
+const tagRef = ref<HTMLDivElement | null>(null)
+const shouldBeHidden = computed(() => {
+  if (tagRef.value == null) return false
```

Comment:
consider `tagRef.value != null && tagRef.value.offsetTop > props.parentOffset`, although which one you use is mostly a matter of preference

Reply:
alternatively, you can do:
```ts
watch(tagRef, tag => {
  if (tag != null && tag.offsetTop > props.parentOffset) {
    emit('hidden')
  }
})
```

Reply:
Leaving it as it is. The second solution won’t work as I need a computed value anyway.